src/client/game.cpp

```diff
@@ -2957,10 +2957,15 @@ void Game::processPlayerInteraction(f32 dtime, bool show_hud, bool show_debug)
 		hlist ? hlist->getItem(0).getDefinition(itemdef_manager) : itemdef_manager->get("");
 
 	v3f player_position  = player->getPosition();
+	v3f player_eye_position = player_position + player->getEyeOffset();
```

Comment:
i think it's better to implement a Player::getEyePosition() and use it here, it can be useless in some other areas :)

Reply:
Done. ```LocalPlayer::getEyePosition()``` is used instead.